index.js

```diff
@@ -0,0 +1,159 @@
+const soap = require('soap');
+const Joi = require('joi');
+const { promisify: prom } = require('util');
+
+let SoapClient;
+let url = 'https://devtest.dotfit.com/webservices/OrdersService.asmx?WSDL';
```

Comment:
we need to be able to change this based on dev/prod

Reply:
https://github.com/firstandthird/node-dotfit/pull/1/files#diff-1e758beff86240736d07379a17316a7bR7